recipes/auxiliary/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "auxiliary" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/auxiliary-{{ version }}.tar.gz
+  sha256: fbc7a5f1bb2043a9fb3cf3ad94411a7e03effecaa53d8208b2dec7c6f6cdb4d5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - poetry-dynamic-versioning >=1.0.0,<2.0.0
+    - pip
+  run:
+    - python >={{ python_min }}
+    # - pathlib >=1.0
+    - nibabel >=3.0
+    - numpy >=1.24
+    - tifffile >=2023.8.25
+    - path >=17.0.0
+    - pillow >=10.0.0
+
+test:
+  imports:
+    - auxiliary.nifti.io
+  # commands:
+  #   - pip check
```

Comment:
Why? If you do this you force all recipes downstream to skip this test too. It is better to fix it if possible.

Reply:
Okay, I am going to try and patch the `pathlib` requirement out. Because it is present in the package's toml file [[ref](https://github.com/BrainLesion/auxiliary/blob/main/pyproject.toml#L20)], `pip check` is complaining.

Reply:
I have also put in a PR to fix this upstream: https://github.com/BrainLesion/auxiliary/pull/21